docs/source/conf.py

```diff
@@ -18,12 +18,8 @@
 # -- Project information -----------------------------------------------------
 
 project = 'ScanCode Workbench Documentation'
-copyright = '2020 nexB Inc.'
-author = 'nexB Inc.'
-
-version = '0.0.1'
-# The full version, including alpha/beta/rc tags
-release = version
+copyright = '2023 nexB Inc.'
```

Comment:
See https://github.com/nexB/skeleton/blob/main/docs/source/conf.py#L20 
We don't want year in the copyright. And others can be included in both copyright and authors.